src/i18n/locales/pl.json

```diff
@@ -165,14 +165,13 @@
       "connectingToBlockchain": "Łączenie z blockchainem NANO",
       "reconnectingToBlockchain": "Rozłączono, ponowne lączenie z blockchainem NANO",
       "cpsLocal": "CPS obserwowany na lokalnym węźle",
-      "cpsNetwork": "Mediany CPS w kilku reprezentatywnych węzłach",
+      "cpsNetwork": "Mediana CPS w kilku reprezentatywnych węzłach",
       "viewSocialEngagement": "Zobacz zaangażowanie społeczne"
     },
     "account": {
       "votingWeight": "Waga głosu",
       "notVoting": "Nie Głosuje",
       "percentNetworkVotingWeight": "% wagi głosu w sieci",
-      "percentRepresentativeVotingWeight": "% wagi głosu przedstawiciela",
```

Comment:
I see that this string is not being used, can you remove it from the other langs as well?